test_runner/src/main/kotlin/ftl/run/platform/android/CreateAndroidTestContext.kt

```diff
@@ -118,7 +118,8 @@ internal fun InstrumentationTestContext.getFlankTestMethods(
     testFilter: TestFilter
 ): List<FlankTestMethod> =
     getParametrizedClasses().let { parameterizedClasses: List<TestMethod> ->
-        DexParser.findTestMethods(test.local).asSequence()
+        DexParser.findTestMethods(test.local, listOf("org.junit.experimental.theories.Theory"))
```

Comment:
Let's have this list as a separate value, we might want to extend it in the future.

WDYT?

Reply:
Do you mean as a parameter a user can pass or just a global const or something?

Reply:
Private value within `CreateAndroidTestContext.kt` file

Reply:
Agreed move it out then we can merge 👍

Reply:
Done!